QUARKUS-874.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-874 Production support for Windows
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-874
+
+The goal is to ensure some level of support and identify existing limitations on Quarkus side for Windows.
+
+## Scope of the testing
+The selected Test Suites will be executed on the selected Windows Platforms to ensure functionality is fully accomplished.
+
+### Selected Windows Platforms
+- Windows Server 2019
+
+### Selected Test Suites
+ - Start-Stop Test Suite
+   - In Github CI - for Pull Requests
+   - New Jenkins Jobs
+ - Beefy Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+   - New Jenkins Jobs
+ - QuickStarts Test Suite
+   - New Jenkins Jobs
+ - Combinations of Quarkus extensions Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+ - Workshop Test Suite
+   - New Jenkins Jobs
+
```

Comment:
Do we need to update OpenShift Test Suite as well? As far I see it, the test suite is not OS dependant.

Reply:
For OpenShift Test Suite we should run some selected modules like the deployment strategies to ensure some functionality baseline checks.

Reply:
Good point. Added.